ArcGISRuntimeSDKQt_CppSamples/Maps/ChangeViewpoint/README.md

```diff
@@ -33,11 +35,7 @@ The map view has several methods for setting its current viewpoint. Select a vie
 
 Below are some other ways to set a viewpoint:
 
-* setViewpoint
-* setViewpointCenter
-* setViewpointGeometry
-* setViewpointRotation
-* setViewpointScale
+* GeoView::setViewpointAsync
```

Comment:
Should we just change the methods to mention "Async"? It does say "Below are some other ways to set a viewpoint:", and it's no longer a list.